Jenkinsfile.deploy

```diff
@@ -66,6 +66,10 @@ pipeline {
                 }
             }
             steps {
+                // Workaround https://issues.apache.org/jira/browse/INFRA-24713
+                def text = readFile ".mvn/wrapper/maven-wrapper.properties"
+                text.replaceAll("distributionUrl=.*", "https://repo.maven.apache.org/maven2/org/apache/maven/apache-maven/3.8.8/apache-maven-3.8.8-bin.zip")
+
```

Comment:
This approach did not work for our CI. The snapshot deployment is failing again: https://ci-builds.apache.org/job/Camel/job/Camel%20Daily%20Snapshot%20Deploy/job/main/363/

Reply:
Sorry about that. Here is an attempt to fix it: https://github.com/apache/camel/pull/10486